host_io/info.json

```
@@ -0,0 +1,216 @@
+{
+    "name": "host_io",
+    "version": "1.0.0",
+    "label": "host.io",
+    "description": "host.io helps to get comprehensive domain name data, uncover new domains and the relationships between them, get DNS details, scraped website content, outbound links, backlinks, and other hosting details for any domain. This connector facilitates automated operation related to various domains.",
+    "publisher": "spryIQ.co",
+    "icon_small_name": "host_io32x32.png",
+    "icon_large_name": "host_io80x80.png",
+    "help_online": "https://github.com/bidisha-spryiq/connector-host_io/blob/develop/docs/HostioConnectorDoc.md",
+    "category": "Data enrichment",
+    "cs_compatible": true,
+    "cs_approved": false,
+    "configuration": {
+        "fields": [
+            {
+                "title": "API Token",
+                "type": "password",
+                "description": "Provide API token, used for user authentication.",
+                "tooltip": "Provide API token, used for user authentication.",
+                "name": "token",
+                "required": true,
+                "visible": true,
+                "editable": true
+            },
+            {
+                "title": "Server URL",
+                "type": "text",
+                "description": "URL of the host.io connector to access the connector website.",
+                "tooltip": "URL of the host.io connector to access the connector website. e.g, https://host.io",
+                "value": "host.io",
+                "placeholder": "host.io or https://host.io",
+                "name": "server",
+                "required": true,
+                "visible": true,
+                "editable": true
+            }
+        ]
+    },
+    "operations": [
+        {
+            "operation": "get_web_domain_details",
+            "title": "Get Web Domain Details",
+            "description": "Get metadata scraped from a domain homepage.",
+            "enabled": true,
+            "category": "utilities",
```

```suggestion
            "category": "investigation",
```
Please update it for all actions.

NOTE:  for actions there are four categories:  1. investigation 2. remediation 3. containment 4. miscellaneous